src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4013,7 +4013,7 @@ namespace SimAirServingZones {
                                    ScalableSM + "Supply Air Flow Rate [m3/s]",
                                    PrimaryAirSystem(AirLoopNum).DesignVolFlowRate);
                 // Initialize MaxOutAir for DOAS loops with no actual OASys, systems with an OA controller will overwrite this is CalcOAController
-                if (PrimaryAirSystem(AirLoopNum).OASysExists)
+                if (PrimaryAirSystem(AirLoopNum).isAllOA)
```

Comment:
Not sure what the difference here is. If it's all OA then isn't OASysExists = true? And if not all OA then OASysExists could be true or false. If false then MaxOutAir = 0 (or there is a DOAS?). So not sure this change has any affect.

Reply:
No, isAllOA means the branch inlet node is an OA node, no OA sys, no OA controller, no return. See 5ZoneAirCooledWithDOASAirLoop.idf.